### PR TITLE
Create channels cleanly & adds icons to claim-list alt-controls

### PR DESCRIPTION
**Pull request by osilkin98**
## Added a 'New Channel' Button

image

## Created a dedicated Channel Creation Page
image

### CODE REVIEW

File: ui/component/channelCreate/view.jsx [hunk 1]
```diff
@@ -0,0 +1,192 @@
+// @flow
```
Comment (seanyesmunt) [inline ui/component/channelCreate/view.jsx]: We already have the `SelectChannel` component. We should be able to re-use most of that code here.

File: ui/page/channels/view.jsx [hunk 2]
```diff
@@ -43,6 +45,15 @@ export default function ChannelsPage(props: Props) {
             header={__('Your Channels')}
             loading={fetchingChannels}
             uris={channels.map(channel => channel.permanent_url)}
+            headerAltControls={
+              <Button
+                iconSize={20}
+                label={__('New Channel')}
+                button="link"
+                icon={ICONS.EDIT}
```
Comment (seanyesmunt) [inline ui/page/channels/view.jsx]: This icon shouldn't be used here. We don't have icons on other links in the header so we don't need one here.

File: ui/page/channels/view.jsx [hunk 3]
```diff
@@ -54,7 +65,13 @@ export default function ChannelsPage(props: Props) {
                 <h2 className="section__title--large">{__('No Channels Created Yet')}</h2>
 
                 <div className="section__actions">
-                  <Button button="primary" navigate="/$/publish" label={__('Create A Channel')} />
+                  <Button
+                    iconSize={20}
+                    label={__('New Channel')}
+                    button="link"
+                    icon={ICONS.EDIT}
```
Comment (seanyesmunt) [inline ui/page/channels/view.jsx]: Same here. This icon isn't for creating a new channel.

**Comment by osilkin98** [comment]
@seanyesmunt `CreateChannel` being on a separate page is only a temporary solution. This is what I had in mind for the final component:
image

I'd like to just get a create channel feature out ASAP as I've had countless people bring it up to me.

**Comment by osilkin98** [comment]
## Channels
image

## Discover
image

## Publishes
image

**Comment by seanyesmunt** [comment]
Closing in favor of #3329

**Pull request closed without merging**